test/models/UserSpec.scala

```diff
@@ -16,10 +16,10 @@
 
 package models
 
+import controllers.ControllerBaseSpec
 import uk.gov.hmrc.auth.core.{Enrolment, EnrolmentIdentifier, Enrolments}
-import uk.gov.hmrc.play.test.UnitSpec
 
-class UserSpec extends UnitSpec {
+class UserSpec extends ControllerBaseSpec {
```

Comment:
Any reason why this and other model specs now inherit `ControllerBaseSpec` instead of the recommended replacement for `UnitSpec`? Just thinking that these specs will now inherit many components they don't need, and could make issues harder to troubleshoot in the future (e.g., something is blowing up the controller tests but not other specs, the issue could lie in the controller base spec).